lib/browser.ts

```diff
@@ -838,7 +838,7 @@ export class ProtractorBrowser extends AbstractExtendedWebDriver {
 
     // Reset bpClient sync
     if (this.bpClient) {
-      await this.bpClient.setWaitEnabled(!this.internalIgnoreSynchronization);
+      await this.bpClient.setWaitEnabled(await this.waitForAngularEnabled());
```

Comment:
I think this change is also in the other PR - it's ok to use internalIgnoreSynchronization here